src/__test_resources__/smallLegacyPivotFolder.ts

```diff
@@ -0,0 +1,180 @@
+/**
+ * The content of the /pivot folder on a Content Server, useful for unit tests.
+ */
+export const smallLegacyPivotFolder = {
```

Comment:
This doesn't seem to be used anywhere right now.
Can you add a test of `migrateUIFolder` that leverages it ?